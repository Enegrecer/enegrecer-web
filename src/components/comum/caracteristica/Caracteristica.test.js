import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Caracteristica from '../caracteristica';

describe('Caracteristica', () => {
  it('deve possuir estrutura definida ', () => {
    const onChange = jest.fn();
    const caracteristica = mount(<Caracteristica id={'1'} onChange={onChange} label="Característica da Vítima" />);
    const caracteristicaTextoJson = toJson(caracteristica);
    expect(caracteristicaTextoJson).toMatchSnapshot();
  });
});

