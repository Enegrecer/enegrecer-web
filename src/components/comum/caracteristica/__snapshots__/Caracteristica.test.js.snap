// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Caracteristica deve possuir estrutura definida  1`] = `
<caracteristica
  id="1"
  label="Característica da Vítima"
  onChange={[Function]}
>
  <campoTexto
    divClasse="input-field col s12"
    id="1"
    inputClasse="materialize-textarea"
    label="Característica da Vítima"
    maxLen={255}
    onChange={[Function]}
    placeholder="Ex.: Era uma mulher negra, com aproximadamente 40 anos, magra, alta com cabelo curto..."
    type="text"
  >
    <div
      className="input-field col s12"
    >
      <input
        className="materialize-textarea"
        id="1"
        maxLength={255}
        onChange={[Function]}
        placeholder="Ex.: Era uma mulher negra, com aproximadamente 40 anos, magra, alta com cabelo curto..."
        type="text"
      />
      <label
        className="active"
        htmlFor="1"
      >
        Característica da Vítima
      </label>
    </div>
  </campoTexto>
</caracteristica>
`;
